modules/core/src/main/java/org/apache/synapse/message/store/impl/jms/JmsStore.java

```diff
@@ -635,7 +558,7 @@ private boolean initme() {
             }
             connectionFactory = lookup(context, javax.jms.ConnectionFactory.class, connectionFac);
             if (connectionFactory == null) {
-                throw new SynapseException(nameString() + " could not initialize JMS Connection Factory. " +
+                throw new StoreForwardException(nameString() + " could not initialize JMS Connection Factory. " +
                                            "Connection factory not found : " + connectionFac);
```

Comment:
Operator should precede the line shall we fix this ?

Reply:
Fixed.